ansible/roles/sufia/defaults/main.yml

```diff
@@ -11,3 +11,8 @@ project_deploy_key: ''
 project_app_root: '{{ project_user_home }}/{{ project_name }}'
 # Name of Solr test core used by application
 project_solr_test_core: 'test'
+# ORCID API endpoints
+project_orcid_site_url: "{{ 'http://api.orcid.org' if project_orcid_service == 'production' else 'http://api.sandbox.orcid.org' }}"
```

Comment:
I also wanted to be explicit in selecting the URLs for the production service so that the failure mode (if `project_orcid_service` wasn't set properly) would be to default to the sandbox service URLs.